src/PowerPing/PingResults.cs

```diff
@@ -50,7 +50,7 @@ public class PingResults
 
         // Local variables
         private readonly Stopwatch operationTimer = new Stopwatch();
-        private ulong sum  = 0; // Sum off all reply times
+        private double sum = 0; // Sum of all reply times
```

Comment:
Increase accuracy of averages - don't round each ping to a whole millisecond first. I noticed this inaccuracy when I was pinging my router (sub-millisecond responses).

Reply:
Yeah makes sense, especially as we are handling doubles in SaveResponsetime good idea to cut out the unneeded casting and loss of accuracy.